src/main/app/src/assets/i18n/nl.json

```diff
@@ -445,6 +445,8 @@
   "indicatie.GEHEIMHOUDING_OP_PERSOONSGEGEVENS": "geheimhouding op persoonsgegevens",
   "indicatie.BLOKKERING_VANWEGE_VERHUIZING": "blokkering vanwege verhuizing",
   "indicatie.OVERLEDEN": "overleden",
+  "indicatie.MINISTIRIELE_REGELING": "ministerieel besluit",
+  "indicatie.EMIGRATIE": "emigratie",
```

Comment:
not wanting to be picky, but I'd rather have these in alphabetical order...  do you agree?